pages/monsters/[id].vue

```diff
@@ -305,4 +354,21 @@ const abilites = [
     width: 100%;
   }
 }
+
+[data-mode='compact'] {
+  font-size: 0.833rem;
+  line-height: 1.25;
+}
+
+[data-mode='compact'] h1 {
+  font-size: 1.44rem;
+}
+
+[data-mode='compact'] h2 {
+  font-size: 1.2rem;
+}
+
+[data-mode='compact'] hr {
+  margin-block: 0.5rem;
+}
```

Comment:
I based the smaller font-sizes on the default type scale at https://typescale.com and I reduced the vertical margins on the <hr>s